app/helpers/session.rb

```diff
@@ -0,0 +1,16 @@
+def current_user
+  session[:user]
+end
```

Comment:
Usually we use `current_user` to be our reference to the actual user object, but it seems like `session[:user]` is just a user name (judging by line 6 of this file).